client/src/components/Icons/CircleTick.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export default ({ color = 'currentColor', size = '3em' }) => (
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" height={size} width={size}>
```

Comment:
I assume that there are no font icons in the admin module we can use for this?

Reply:
Only one of the two (the tick) has an equivalent in the admin icon font, and it's not quite the same. For consistency in implementation between the two, I'd like to leave this as-is.